operators/src/processing/meteosat/temperature.rs

```diff
@@ -302,9 +301,13 @@ where
         query: RasterQueryRectangle,
         ctx: &'a dyn QueryContext,
     ) -> Result<BoxStream<'a, Result<Self::Output>>> {
-        let src = self.source.query(query, ctx).await?;
-        let rs = src.and_then(move |tile| self.process_tile_async(tile, ctx.thread_pool().clone()));
-        Ok(rs.boxed())
+        stack_individual_aligned_raster_bands(&query, ctx, |query, ctx| async move {
```

Comment:
why is this needed for this operator but not for the other MSG Operators?

Reply:
I initially thought it was needed for all msg operators, but as they just map tiles and don't care about the band, I removed it again. I only forgot it here: 02d77a9e4c6b54b8a0ee8a272c89873799792d19